From 08e3daec6486fd6b4edbecb0ee605d4e1e2cff05 Mon Sep 17 00:00:00 2001
Message-Id: <08e3daec6486fd6b4edbecb0ee605d4e1e2cff05.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Damien Lespiau <damien.lespiau@intel.com>
Date: Sat, 30 Aug 2014 16:51:02 +0100
Subject: [PATCH 278/312] FROM_UPSTREAM [VPG]: drm/i915: Remove unneeded
 brackets

Issue: VIZ-4092
Issue: GMIN-3475
Upstream-repo: git://anongit.freedesktop.org/drm-intel
Upstream-commit: b07ba1dc78a251fc02992a35b0fd8757029566e4
Change-Id: Icebfe22e9057987482e88723e11cdec9afe66586
Signed-off-by: Damien Lespiau <damien.lespiau@intel.com>
Reviewed-by: Arun Siluvery <arun.siluvery@linux.intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Signed-off-by: Michel Thierry <michel.thierry@intel.com>
---
 drivers/gpu/drm/i915/intel_ringbuffer.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_ringbuffer.c b/drivers/gpu/drm/i915/intel_ringbuffer.c
index 5a889cd..2de5c67 100644
--- a/drivers/gpu/drm/i915/intel_ringbuffer.c
+++ b/drivers/gpu/drm/i915/intel_ringbuffer.c
@@ -622,7 +622,7 @@ static inline void intel_ring_emit_wa(struct intel_engine_cs *ring,
 	intel_ring_emit(ring, value);
 
 	dev_priv->intel_wa_regs[dev_priv->num_wa_regs].addr = addr;
-	dev_priv->intel_wa_regs[dev_priv->num_wa_regs].mask = (value) & 0xFFFF;
+	dev_priv->intel_wa_regs[dev_priv->num_wa_regs].mask = value & 0xFFFF;
 	/* value is updated with the status of remaining bits of this
 	 * register when it is read from debugfs file
 	 */
-- 
1.7.9.5

